sqlglot/expressions.py

```diff
@@ -5325,6 +5325,7 @@ class Unnest(Func, UDTF):
         "expressions": True,
         "alias": False,
         "offset": False,
+        "nested_array": False,
```

Comment:
The intent isn't very clear with this name imo, consider renaming to "explode_array" or something similar.